src/messagecard.ts

```diff
@@ -9,38 +9,110 @@ export function buildMessageCard(
   repoBranch: string,
   avatar_url: string,
   login: string,
-  author_url: string
+  author_url: string,
+  link: string
 ): string {
-  const card = {
-    '@type': 'MessageCard',
-    '@context': 'https://schema.org/extensions',
-    $schema: 'https://adaptivecards.io/schemas/adaptive-card.json',
-    version: '1.0',
-    summary: messageTitle,
-    themeColor: messageColour,
-    title: messageTitle,
-    sections: [
+  let actions: Record<string, string>[] = []
```

Comment:
Can this be a const?

Reply:
The actions variable is either an empty list or it is the full element, hence not being a const.

Reply:
I didn't clock the reassignment on line 17